gcloud/streaming/util.py

```diff
@@ -9,6 +8,21 @@
 
 
 def type_check(arg, arg_type, msg=None):
+    """Assert that arg is of the expected type.
```

Comment:
Do we have a plan for murdering this function? Is it actually necessary? File a bug?